src/web/lib/ArticleRenderer.tsx

```diff
@@ -519,6 +522,75 @@ export const ArticleRenderer: React.FC<{
 							language={element.language}
 						/>
 					);
+				case 'model.dotcomrendering.pageElements.WitnessBlockElement': {
+					const witnessType = element.witnessTypeData._type;
+					switch (witnessType) {
+						case 'model.dotcomrendering.pageElements.WitnessTypeDataImage':
+							const witnessTypeDataImage = element.witnessTypeData as WitnessTypeDataImage;
+							return (
+								<figure itemType="http://schema.org/ImageObject">
```

Comment:
We should be using `Figure` here. That will set the top and bottom margins which don't seem to exist here